index.html

```diff
@@ -266,36 +276,58 @@ <h3 class="ctitletext">
           to chat please get in touch.
         </h3>
       </div>
-      <form class="contactform">
-        <input type="text" name="name" placeholder="Full name" class="cdata" />
+      <form class="contactform"  action="https://formspree.io/f/xlezzayo"
+        method="POST">
+>
         <input
           type="text"
+          name="name"
+          maxlength="30"
+          placeholder="Full name"
+          class="cdata"
+          required
+        />
+        <input
+          type="email"
           name="email"
           placeholder="Email address"
           class="cdata"
+          required
         />
         <textarea
           name="message"
           placeholder="Enter Text Here"
           class="cmessage"
+          maxlength="500"
+          required
         ></textarea>
         <input class="csubmit" type="submit" value="Get in touch" />
```

Comment:
- Please add button interaction as it is shown in the Figma, such as hover, active (when it's pressed), disabled and enable (default state)